### ISSUE TITLE
MAPELoss description is wrong

### ISSUE THREAD

**Issue by squall1988** [tensorflow/tensorflow#39867]
https://github.com/tensorflow/tensorflow/blob/2b96f3662bd776e277f86997659e61046b56c315/tensorflow/python/keras/losses.py#L1242

MAPELoss description is wrong, it's not 
loss = 100 * mean(abs(y_true - y_pred) / y_true, axis=-1)
but may be this:
loss = 100*mean(abs((y_true - y_pred)/y_true, axis=-1))

**Comment by jvishnuvardhan** [comment]
@squall1988 Looks like only description is not correct as you mentioned. But the formula used in the code is correct. 

Are you interested in raising PR to correct the doc? Thanks!
